### PR TITLE
Google.AndroidM5Emulator version m5-rc14

### PR DESCRIPTION
**Pull request by upintheairsheep**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/106157)

### CODE REVIEW

**Comment by stephengillie** [comment]
Automatic Validation ended with:
InstallationVerification-x86-Scope_Undefined-Locale_Undefined-ErrorScreenshot_20230507201234

File: manifests/g/Google/AndroidM5Emulator/m5-rc14/Google.AndroidM5Emulator.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Google.AndroidM5Emulator
+PackageVersion: m5-rc14
+Installers:
+- InstallerUrl: https://dl.google.com/android/archives/android-sdk_m5-rc14_windows.zip
+  Architecture: x86
+  InstallerType: zip
+  NestedInstallerType: exe
```
[the remark this answers is not in the document]
Comment (stephengillie) [reply manifests/g/Google/AndroidM5Emulator/m5-rc14/Google.AndroidM5Emulator.installer.yaml:10]: This package has the same situation as the [M3 emulator](https://github.com/microsoft/winget-pkgs/pull/106155) in that the direct package link works, but the symlink is expecting an argument with the directory:
```
Cannot locate system directory, which contains 'system.img' and other
system files.  Please specify one by using '-system <directory>' or by
defining the environment variable ANDROID_PRODUCT_OUT.
```

**Comment by stephengillie** [comment]
https://github.com/microsoft/winget-pkgs/pull/106157#discussion_r1187737350

**Comment by ItzLevvie** [comment]
[Policy] Blocking-Issue

**Comment by Trenly** [comment]
@upintheairsheep - Support for zipped binaries has now been added and is available in the pipelines. This will require you to update your submission to ManifestVersion: 1.9.0 and set ArchiveBinariesDependOnPath: true in the installer node. To keep things clean, this PR is being closed. Please submit a new PR for the latest version of this package. Thank you for your contributions!

**Pull request closed without merging**